src/app/components/article/ArticleTrash.js

```diff
@@ -20,6 +21,11 @@ const updateExplainer = graphql`
         id
         trashed
       }
+      team {
+        articles_count
```

Comment:
Is this generic `articles_count` still needed now that we have aliases for each list?

Reply:
No, you're right, removed.